include/vctr/Containers/VctrBase.h

```diff
@@ -20,6 +20,11 @@
   ==============================================================================
 */
 
+#if VCTR_MSVC
+__pragma(warning(push))
+    __pragma(warning(disable:4324))
```

Comment:
Funky indentation..?

Reply:
I'd rather wrap that in clang-format guards and align it manually, don't see any good reason why this should look like this, I rather consider this a clang-format fault